### PR TITLE
Feature/numeric state condition

### PR DESCRIPTION
**Pull request by Lukyio**

### CODE REVIEW

**Review by souziel — CHANGES_REQUESTED** [review]
Can you use the i18n for strings in the code please?

**Comment by Lukyio** [comment]
> 
> 
> Can you use the i18n for strings in the code please?

Which line for example ?

File: client/src/routes/automation/esidomBlocks.ts [hunk 1]
```diff
@@ -158,9 +160,76 @@ export type BlocksDefinitions = {
...
+    block.sun_trigger = {
+        init() {
+            this.jsonInit?.(
+                {
+                    type: 'sun_trigger',
+                    message0: '%1 h %2 m %3 s %4 le %5 du soleil',
+                    args0: [
+                        {
+                            type: 'field_number',
+                            name: 'Hour',
```
Comment (souziel) [inline client/src/routes/automation/esidomBlocks.ts:173]: here i18n

File: client/src/services/blocklyService.ts [hunk 2]
```diff
@@ -110,21 +111,190 @@ export default class BlocklyService {
...
+                            {
+                                type: 'input_dummy',
+                                name: 'condition',
+                            },
+                        ],
+                        inputsInline: false,
+                        previousStatement: 'Condition',
+                        nextStatement: 'Condition',
+                        colour: COLORS.HUE_YELLOW,
+                        tooltip: 'Après avoir fourni un capteur numérique, indiquez les valeurs minimum et maximum entre lesquelles le bloc doit/ne doit pas réagir',
```
Comment (souziel) [inline client/src/services/blocklyService.ts]: here i18n

**Comment by Nailykk** [comment]
> 
> 
> Can you use the i18n for strings in the code please?

Done (and more)

**Review by souziel — APPROVED** [review]

**Review by Nailykk — APPROVED** [review]

**Pull request merged**